File: fmxtodo/MainForm.pas
unit MainForm;

interface

uses
  System.SysUtils, System.Types, System.UITypes, System.Classes, System.Variants,
  FMX.Types, FMX.Controls, FMX.Forms, FMX.Graphics, FMX.Dialogs, FMX.StdCtrls,
  FMX.Edit, FMX.Controls.Presentation, FMX.Menus, FMX.Layouts, FMX.ListBox;

type
  TFMainForm = class(TForm)
    Label1: TLabel;
    itemEdit: TEdit;
    addButton: TButton;
    todoList: TListBox;
    todoListMenu: TPopupMenu;
    deleteItem: TMenuItem;
    procedure addButtonClick(Sender: TObject);
    procedure deleteItemClick(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  FMainForm: TFMainForm;

implementation

{$R *.fmx}

procedure TFMainForm.addButtonClick(Sender: TObject);
begin
//
end;

procedure TFMainForm.deleteItemClick(Sender: TObject);
begin
//
end;

end.

File: fmxtodo/MainForm.py
import os
from delphifmx import *

class FMainForm(Form):

    def __init__(self, owner):
        self.Label1 = None
        self.itemEdit = None
        self.addButton = None
        self.todoList = None
        self.todoListMenu = None
        self.deleteItem = None
        self.LoadProps(os.path.join(os.path.dirname(os.path.abspath(__file__)), "MainForm.pyfmx"))

    def addButtonClick(self, Sender):
        pass

    def deleteItemClick(self, Sender):
        pass